app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -99,6 +112,9 @@
     @MetaField(META_SUBMISSION_ORDERING_NUMBER)
     private String submissionOrderingNumber;
 
+    @Persisting(value = 9, nullable = true)

```

Comment:
Confirming: a migration isn't necessary because `nullable` is true

Reply:
Right, the `nullable` makes it not necessary, and there is no other reason to do one, since we would have to make a random guess as to the reason for pre-existing records.

Reply:
I actually didn't realize that it was true that setting nullable made these not require migration, and find it kind of surprising. How does the processor know to stop reading values? If someone added a tenth value after this one and the phone jumped from 2.36 -> That Version I'm not sure I understand technically how it would know to figure out when to stop reading the stream... (Or is it that you can now extend perisistables all you want as long as all values are nullable after the last hard migraiton?)

Reply:
Errr... I was making an assumption, which now that you say things that way, I think must be wrong. I'll add in a migration. sorry about that :/